### PR TITLE
Fixed readme file

### PR DESCRIPTION
**Pull request by lavinan26**

### CODE REVIEW

File: oada-formats-viewer/src/Views/Dropdown.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+      style={{ width: 250 }}
+      renderInput={params => (
+        <TextField {...params} label="Combo box" variant="outlined" fullWidth />
+      )}
+    />
+  );
+}
+
+// Top 100 films as rated by IMDb users. http://www.imdb.com/chart/top
+const top100Films = [
```
Comment (PivithuruThejan) [inline oada-formats-viewer/src/Views/Dropdown.js]: do we need these?

Comment (lavinan26) [reply oada-formats-viewer/src/Views/Dropdown.js]: I fixed this

File: oada-formats-viewer/config/webpack.config.dev.js [hunk 2]
```diff
@@ -0,0 +1,211 @@
+'use strict';
```
Comment (PivithuruThejan) [inline oada-formats-viewer/config/webpack.config.dev.js]: do we need all these?

File: oada-formats-viewer/config/webpack.config.prod.js [hunk 3]
```diff
@@ -0,0 +1,245 @@
+'use strict';
```
Comment (PivithuruThejan) [inline oada-formats-viewer/config/webpack.config.prod.js]: Do we need all these?

File: oada-formats-viewer/config/webpack.config.dev.js (hunk 2 above)
Comment (lavinan26) [reply oada-formats-viewer/config/webpack.config.dev.js]: deleted

File: oada-formats-viewer/config/webpack.config.prod.js (hunk 3 above)
Comment (lavinan26) [reply oada-formats-viewer/config/webpack.config.prod.js]: deleted

**Pull request merged**